site/src/pages/release-notes/1.26.4.mdx

```diff
@@ -0,0 +1,20 @@
+---
+date: 2023-11-2x
+---
+
+## 🛠️ Bug fixes
+
+- Fixed a bug where a <type://RequestLog> isn't complete when an <type://HttpResponse> is cancelled. #5312
+- [gRPC HTTP/JSON transcoding](https://cloud.google.com/endpoints/docs/grpc/transcoding) service now sends
+  400 Bad Request when the request body is invalid JSON. #5306 #5307
+
+## 🙇 Thank you
+
+<ThankYou usernames={[
+  'ikhoon',
+  'jrhee17',
```

Comment:
I'd like to give a credit to @ks-yim who reported the bug of #5312 and the reproducer.
```suggestion
  'jrhee17',
  'ks-kim',
```

Reply:
Added. 😉